inc/classes/optimization/class-cache-dynamic-resource.php

```diff
@@ -179,10 +179,12 @@ public function is_excluded_file( $src ) {
 			return true;
 		}
 
-		$file['query'] = remove_query_arg( 'ver', $file['query'] );
+		if ( is_array( $file ) && ! empty( $file['query'] ) ) {
```

Comment:
Here we can use `isset( $file['query'] )` instead. But we can also simplify this code a little bit more to eliminate by converting this into a guard clause instead. If it's not set, return false; else, remove the `ver` query arg and recheck.


```php
if ( ! isset( $file['query'] ) ) {
	return false;
}

$file['query'] = remove_query_arg( 'ver', $file['query'] );
// ... reset of the code here.
```